controller/controller_server/messages.py

```diff
@@ -3,8 +3,12 @@
 ObjectIdError_message = "Wrong {0} id format : {1}"
 
 # validation error messages
-invalid_secret_message = "invalid secret was passed"
-secret_missing_message = 'secret is missing'
+invalid_secrets_message = "invalid secrets was passed"
```

Comment:
please replace the usages of this generic message with more informative messages, according to each specific issue with the secret, and try to keep it singular

Reply:
done